client/js/components/main/assessment.jsx

```diff
@@ -15,28 +15,32 @@ const select = (state, props) => {
   return {
 
     // Assessment configuration settings. these should never be modified.
-    settings             : state.settings,
+    settings        : state.settings,
 
     // Assessment to be rendered.
-    assessment           : state.assessment,
+    assessment      : state.assessment,
 
     // State of user-assessment interactions.
-    progress             : state.progress.toJS(),
+    progress        : state.progress.toJS(),
 
-    // The index of the current question within the allQuestions array
-    currentQuestion      : state.progress.get('currentItemIndex'),
+    // Current page of items to display when paging through items
+    currentItem : state.progress.get('currentItemIndex'),
 
     // Array of user responses
-    responses            : state.progress.get('responses').toJS(),
+    responses       : state.progress.get('responses').toJS(),
+
+    // How many questions to display at a time. Default to show all questions
+    // in a section if not specified
+    displayNum      : state.settings.questions_per_section || questionCount(state, props),
```

Comment:
I would keep the name the same throughout the app. So rather than displayNum set it to questionsPerSection